client/app/assets/less/redash/ant.less

```diff
@@ -36,3 +42,35 @@
 .ant-dropdown-in-bootstrap-modal {
   z-index: 1050;
 }
+
+// Make Ant labels more Redash like
+.ant-form-item-label label {
+  color: inherit;
+  white-space: initial;
+  line-height: initial;
+  text-align: left;
```

Comment:
You might want to either use `direction: rtl` instead (to flip the required mark location) or maybe ant-form's `layout="vertical"` (didn't try but sounds like it might work).

Reply:
Gotcha. I agree. 
Add this attribute `<Form layout="vertical">` and you won't need a css override.